src/TouristInfoCards.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import GlasgowImg from "./images/George Square Glasgow.jpeg";
+import LancasterCastleImg from "./images/Lancaster Castle.jpeg";
+import LondonEyeImg from "./images/London Eye.jpeg";
+
+const TouristInfoCards = () => {
```

Comment:
This is all ok. But, you can make it nicer ; )
Please create an array of objects. Each object would present each city and then map on your data to display them in return.